app/controller/auth.js

```diff
@@ -52,42 +56,52 @@ app.get('/logout', function(req, res) {
   res.redirect('/');
 });
 
-// Export some convienience functions
+// Convienience functions for authentication in express
 export function loggedIn(req, res, next) {
   if (req.isAuthenticated()) {
     return next();
   }
+
   return res.redirect('/auth/github');
 }
 
 export function isBeta(req, res, next) {
```

Comment:
I think I might leave that for another PR. I also want to setup page redirection so login doesn't always take you to dashboard (helpful for if we add another page), and add some GitHub function to check if the user is a contributor for a repo (so people can't build other people's stuff).